lib/elixir/lib/config.ex

```diff
@@ -160,7 +160,8 @@ defmodule Config do
   end
 
   @doc """
-  Returns the environemnt this configuration file is executed on.
+  In Mix projects this function returns the environment this configuration
```

Comment:
The first line should always be a short summary, as that is shown in tables and stuff, so what do you think about this?

```suggestion
  Returns the environment this configuration file is executed on.

  In Mix projects this function returns the environment this configuration
```

Reply:
Ah, forgot that! Sure, let me rebase in a while!

Reply:
There is no need to rebase, we always squash it anyway. :) Thanks!